### ISSUE TITLE
Bug: Throwing an error when fetching the email avatar

### ISSUE THREAD

**Issue by avarayr** [Foundry376/Mailspring#1909]
**Describe the bug**
An exception is thrown when loading the email's avatar.

**To Reproduce**
Try to load the user email avatar

**Expected behavior**
An avatar is shown.

**Screenshots**
![image](https://user-images.githubusercontent.com/7735415/79513667-3de19680-7ff9-11ea-8f3f-f5c1115006f7.png)

**Full stack trace**
```javascript
TypeError: Cannot read property 'startsWith' of null
    at SidebarParticipantProfile._renderPersonInfo (file:///var/folders/nz/vv4_9tw56nv9k3tkvyszvwg80000gn/T/nylas-build/electron-packager/darwin-x64/Mailspring-darwin-x64/Electron.app/Contents/Resources/app/internal_packages/participant-profile/lib/sidebar-participant-profile.tsx:491:48)
    at SidebarParticipantProfile.render (file:///var/folders/nz/vv4_9tw56nv9k3tkvyszvwg80000gn/T/nylas-build/electron-packager/darwin-x64/Mailspring-darwin-x64/Electron.app/Contents/Resources/app/internal_packages/participant-profile/lib/sidebar-participant-profile.tsx:513:15)
    at ah (/Applications/Mailspring.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:163:192)
    at $g (/Applications/Mailspring.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:162:264)
    at eh (/Applications/Mailspring.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:171:13)
    at ai (/Applications/Mailspring.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:202:255)
    at bi (/Applications/Mailspring.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:203:271)
    at Fi (/Applications/Mailspring.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:218:174)
    at Gi (/Applications/Mailspring.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:217:174)
    at ii (/Applications/Mailspring.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:214:252)
    at xg (/Applications/Mailspring.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:212:94)
    at Object.enqueueSetState (/Applications/Mailspring.app/Contents/Resources/app.asar/node_modules/react-dom/cjs/react-dom.production.min.js:133:223)
    at FocusedContactStorePropsContainer.Component.setState (/Applications/Mailspring.app/Contents/Resources/app.asar/node_modules/react/cjs/react.development.js:373:16)
    at FocusedContactsStore.FocusedContactStorePropsContainer._onChange (file:///var/folders/nz/vv4_9tw56nv9k3tkvyszvwg80000gn/T/nylas-build/electron-packager/darwin-x64/Mailspring-darwin-x64/Electron.app/Contents/Resources/app/internal_packages/message-list/lib/sidebar-plugin-container.tsx:27:10)
    at EventEmitter.eventHandler (file:///var/folders/nz/vv4_9tw56nv9k3tkvyszvwg80000gn/T/nylas-build/electron-packager/darwin-x64/Mailspring-darwin-x64/Electron.app/Contents/Resources/app/src/global/mailspring-store.ts:128:23)
    at EventEmitter.emit (events.js:187:15)
    at EventEmitter.emit (domain.js:442:20)
    at FocusedContactsStore.trigger (file:///var/folders/nz/vv4_9tw56nv9k3tkvyszvwg80000gn/T/nylas-build/electron-packager/darwin-x64/Mailspring-darwin-x64/Electron.app/Contents/Resources/app/src/global/mailspring-store.ts:141:26)
    at later (/Applications/Mailspring.app/Contents/Resources/app.asar/node_modules/underscore/underscore.js:828:25)
```

**Setup:**
- OS: Mac OS Catalina
 - Mailspring Version: 1.7.5

**Additional context** 

The email whose avatar is being loaded has a Google+ avatar set.

**Comment by bengotow** [comment]
Ahh man that's frustrating, it looks like the TypeScript types for the data returned from the server weren't quite right and I assumed they were. I think that we can patch this on the backend as well to avoid having to ship another update today. Will do that as well.

**Comment by bengotow** [comment]
Ok I've patched this on the server as well—it should disappear soon and will be permanently fixed in the next release. Thanks!

**Comment by bengotow** [comment]
Hey folks, thanks for reporting this. Happy to say this has been resolved in 1.7.6 which shipped this afternoon. 🚀  The app should auto-update, but if it doesn't you can also grab the new release from GitHub's Releases tab or from https://getmailspring.com/download. Thanks for your patience and help tracking this down, enjoy the update!
